pkg/kubernetes/scripts/policy.js

```diff
@@ -179,18 +179,40 @@
                     kind: "RoleBinding",
                     apiVersion: "v1",
                     metadata: {
-                        name: name,
+                        name: role,
                         namespace: namespace,
-                        subjects: subjects,
+                        creationTimestamp: null,
                     },
+                    userNames: [],
+                    groupNames: [],
+                    subjects: [],
                     roleRef: {
-                        name: name
+                        name: role
                     }
                 };
-
-                return methods.create([binding], namespace);
+                addToArray(roleArray(binding, "subjects"), subjects);
+                addToArray(roleArrayKind(binding, subjects.kind), subjects.name);
+                return methods.create(binding, namespace);   
             }
+            function removeMemberFromPolicyBinding(policyBinding, project, subjectRoleBindings, subject) {
+                var roleBinding, i, defaultPolicybinding;
+                var roleBindings = [];
 
+                if(policyBinding && policyBinding.one()){
+                    defaultPolicybinding = policyBinding.one();
+                    roleBindings = defaultPolicybinding.roleBindings;
+                }
+                var patchData = {"roleBindings": roleBindings};
+                angular.forEach(subjectRoleBindings, function(o) {
+                    for (i = roleBindings.length - 1; i >= 0; i--) {
```

Comment:
i should declared inside this function. Or better yet, just use angular.forEach().